cli/src/client/mod.rs

```diff
@@ -0,0 +1,44 @@
+use graphql_client::{GraphQLQuery, Response};
+use http::Uri;
+use reqwest::blocking;
+use std::error::Error;
+
+mod queries;
+
+static PROD_GQL_API_URL: &str = "https://engine-graphql.apollographql.com/api/graphql";
+
+pub struct Client {
+    api_key: String,
+    uri: Uri,
+    reqwest: blocking::Client,
+}
+
+impl Client {
+    fn from(api_key: String) -> Client {
+        Client {
+            api_key,
+            uri: PROD_GQL_API_URL.parse::<Uri>().unwrap(),
+            reqwest: blocking::Client::new(),
+        }
+    }
+
+    fn send<Q: GraphQLQuery>(
+        &self,
+        variables: Q::Variables,
+    ) -> Result<Response<Q::ResponseData>, Box<dyn Error>> {
+        let request_body = Q::build_query(variables);
+
+        let res = self
+            .reqwest
+            .post(&self.uri.to_string())
+            .header("Content-Type", "application/json")
+            .header("apollographql-client-name", "internal-apollo-cli")
```

Comment:
Do we want to treat this as a new version of the [Apollo CLI](https://github.com/apollographql/apollo-tooling/blob/master/packages/apollo/src/Command.ts#L229-L233) or a new product all together?

Reply:
No preference, whatever you think is best..

Reply:
🤔 I think it would be cool to keep it the same and watch the stats roll over in Graph Manager!